examples/containers/.env

```diff
@@ -0,0 +1,3 @@
+COUCHBASE_SERVER_IMAGE=couchbase/server:7.0.2
+FLUENT_BIT_IMAGE=couchbase/fluent-bit:1.1.2
+CMOS_IMAGE=registry.gitlab.com/cb-vanilla/observability-stack:latest
```

Comment:
Not sure how I feel about defaulting to this, since it'll require you to have the GitLab credentials (I'm not sure if they're listed somewhere people can access) - though I'd imagine that it'll give you a sensible error message if you're not logged in. The other option until the image is on Docker Hub is `couchbase/observability-stack`, which will fail if it doesn't exist locally, which isn't great either. No good options here 😕

Reply:
No, and this is in preparation for the docs updates which will all be using the official ones. We probably should be changing everything to the official DockerHub ones with just a bit of guidance about building them first for now.